packages/framework-core/src/decorators/sequenced-by.ts

```diff
@@ -0,0 +1,21 @@
+import { Booster } from '../booster'
+import { Class, ReadModelInterface } from '@boostercloud/framework-types'
+import { getFunctionArguments } from './metadata'
+
+export function sequencedBy(klass: Class<ReadModelInterface>, _functionName: string, parameterIndex: number): void {
```

Comment:
Maybe this should be  `@SequencedBy`? For some reason camelCase looks weird to me here...